modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/rel/ScannableTableSelfTest.java

```diff
@@ -306,6 +308,8 @@ public void testIndexScanError(NoOpTransaction tx) {
     @ParameterizedTest
     @MethodSource("transactions")
     public void testIndexScanInvalidCondition(NoOpTransaction tx) {
+        assumeTrue(IgniteUtils.assertionsEnabled(), "the test requires that assertions be enabled");
```

Comment:
Do we get any meaningful exception when assertions are disabled?
I think we should throw `InvalidStateException` instead.

Reply:
fixed, thanks